apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonVariantTestSection.tsx

```diff
@@ -1,20 +1,56 @@
 import { Button, CompoundButton } from '@fluentui-react-native/experimental-button';
 import * as React from 'react';
-import { View } from 'react-native';
+import { View, Platform, TouchableOpacity } from 'react-native';
 import { commonTestStyles, stackStyle } from '../Common/styles';
 
 export const ButtonVariantTest: React.FunctionComponent<{}> = () => {
-  return (
-    <View style={[stackStyle, commonTestStyles.view]}>
-      <Button content="Default" style={commonTestStyles.vmargin} />
-      <Button primary content="Primary" style={commonTestStyles.vmargin} />
-      <Button ghost content="Ghost" style={commonTestStyles.vmargin} />
-      <Button fluid content="Fluid" style={commonTestStyles.vmargin} />
-      <Button primary fluid content="Fluid Primary" style={commonTestStyles.vmargin} />
-      <Button ghost fluid content="Fluid Ghost" style={commonTestStyles.vmargin} />
-      {/* <CompoundButton content="Default" secondaryContent="Compound" style={commonTestStyles.vmargin} />
-      <CompoundButton primary content="Primary" secondaryContent="Compound" style={commonTestStyles.vmargin} /> */}
-      <CompoundButton ghost content="Ghost" secondaryContent="Compound" style={commonTestStyles.vmargin} />
-    </View>
-  );
+  const [buttonTitle1, setTitle1] = React.useState('Mac Native Button');
+  const onPress1 = () => setTitle1('Native Button Clicked');
+
+  const [buttonTitle2, setTitle2] = React.useState('Mac Native Button');
+  const onPress2 = () => setTitle2('Native Button Clicked');
+
+  const [buttonTitle3, setTitle3] = React.useState('Mac Native Button');
+  const onPress3 = () => setTitle3('Native Button Clicked');
+
+  if (Platform.OS == 'macos') {
+    return (
+      <View style={[stackStyle, commonTestStyles.view]}>
+        <TouchableOpacity onPress={onPress1}>
```

Comment:
Why do we need to wrap these in TouchableOpacity entries? Doesn't the button support onPress?

Reply:
Yeah, this is sort of a hack to get the onPress event working. I was thinking the proper way of doing it is to export the native onPress event to JS using RCTEventEmitter?

Reply:
I removed the TouchableOpacity wrappers and used RCTBubblingEventBlock to fire events from native.